.circleci/config.yml

```diff
@@ -1,25 +1,21 @@
 version: 2
 jobs:
   build:
-    working_directory: /go/src/github.com/mozilla/sops
+    working_directory: /go/src/go.mozilla.org/sops
     docker:
       - image: circleci/golang:1.8
     steps:
       - checkout
+      - setup_remote_docker
       - run:
-          name: Build Sops
-          command: make 
-      - setup_remote_docker                                                                                                                                                                                                                                    
-      - run:                                                                                                                                                                                                                                                   
-          name: Build containers                                                                                                                                                                                                                               
-          command: |                                                                                                                                                                                                                                           
-            docker build -t mozilla/sops .                                                                                                                                                                                                          
-            docker tag mozilla/sops "mozilla/sops:$CIRCLE_SHA1"                                                                                                                                                                          
+          name: Build containers
+          command: |
+            docker build -t mozilla/sops .
+            docker tag mozilla/sops "mozilla/sops:$CIRCLE_SHA1"
       - run:
           name: Push containers
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
                 ./bin/ci/deploy_dockerhub.sh "latest"
                 ./bin/ci/deploy_dockerhub.sh "$CIRCLE_SHA1"
```

Comment:
This is going to give you a newly tagged image on dockerhub for _every commit_ to master. I suspect you do not want this for something like sops.

Reply:
I think that's fine. We could also tag images with git tags but I don't think pushing on every commit to master would be terrible.